index.html

```diff
@@ -3131,12 +3147,84 @@ <h2>
           readonly attribute DOMString? payerPhone;
 
           Promise&lt;void&gt; complete(optional PaymentComplete result = "unknown");
+          Promise&lt;void&gt; retry();
         };
       </pre>
       <p class="note">
         A <a>PaymentResponse</a> is returned when a user has selected a payment
         method and approved a payment request.
       </p>
+      <section>
+        <h2>
+          <dfn>retry()</dfn> method
+        </h2>
+        <p data-tests="payment-response/retry-method-manual.https.html">
+          The <a>retry()</a> method MUST act as follows:
+        </p>
+        <ol class="algorithm">
+          <li>Let <var>response</var> be the <a>context object</a>.
+          </li>
+          <li>Let <var>request</var> be
+          <var>response</var>.<a>[[\request]]</a>.
+          </li>
+          <li>If <var>response</var>.<a>[[\complete]]</a> is true, return <a>a
+          promise rejected with</a> an "<a>InvalidStateError</a>"
+          <a>DOMException</a>.
+          </li>
+          <li>If <var>response</var>.<a>[[\retrying]]</a> is true, return <a>a
+          promise rejected with</a> an "<a>InvalidStateError</a>"
+          <a>DOMException</a>.
+          </li>
+          <li>Set <var>response</var>.<a>[[\retrying]]</a> to true.
+          </li>
+          <li>Set <var>request</var>.<a>[[\state]]</a> to "<a>interactive</a>".
+          </li>
+          <li>Let <var>retryPromise</var> be <a>a new promise</a>.
+          </li>
+          <li>Set <var>response</var>.<a>[[\retryPromise]]</a> to
+          <var>retryPromise</var>.
+          </li>
+          <li>
+            <a>In parallel</a>:
+            <ol>
+              <li>In the payments UI, indicate to the end-user that something
+              is wrong with the user-provided data of the payment response.
+              </li>
+              <li>
+                <p>
```

Comment:
The first paragraph here doesn't seem like a numbered step. I would put it as a NOTE inside the step 10, "return retryPromise".